fastdeploy/worker/worker_process.py

```diff
@@ -688,6 +697,26 @@ def initialize_fd_config(args, ranks: int = 1, local_rank: int = 0) -> FDConfig:
             parallel_config.local_data_parallel_id
         ]
     parallel_config.set_communicate_group()
+    # config for Attention TP + MoE EP
+    if parallel_config.enable_tensor_or_expert_parallel:
+        expert_parallel_rank = int(local_rank % parallel_config.tensor_parallel_size)
+        if isinstance(model_config.moe_num_experts, list):
+            num_experts = model_config.moe_num_experts[0]
+        else:
+            num_experts = model_config.moe_num_experts
+
+        num_experts_per_rank = num_experts // parallel_config.tensor_parallel_size
+        num_experts_start_offset = expert_parallel_rank * num_experts_per_rank
+        max_chips_per_node = 16 if current_platform.is_iluvatar() else 8
+
+        parallel_config.expert_parallel_rank = expert_parallel_rank
+        parallel_config.num_experts_per_rank = num_experts_per_rank
+        parallel_config.num_experts_start_offset = num_experts_start_offset
+
+    parallel_config.engine_worker_queue_port = parallel_config.engine_worker_queue_port[
+        parallel_config.local_data_parallel_id
+    ]
```

Comment:
All CI fails at this line. ```TypeError: '\''int'\'' object is not subscriptable' ```. We need to solve it first and then see if there are any other problems